frontend/src/general/simpleTable/SimpleTableUtil.js

```diff
@@ -0,0 +1,8 @@
+export function zipProperties(propertyNames, properties) {
+  const arr = [];
+  for (let i = 0; i < propertyNames.length; i += 1) {
+    arr[i] = [propertyNames[i], properties[i]];
+  }
+
+  return arr;
```

Comment:
You can simplify this!

Pass the properties object plainly into SimpleTable, instead of passing 2 arrays. Example:
```
// In StreamDeviceDetails.jsx
const properties = {
    "Name": device.name,
    "Serial Number": device.serialNumber,
    "Channel": channel
};
```
Then, take just the properties object in this zipProperties method, and do
```
// In SimpleTableUtil.js
return Object.keys(properties).map((name) => {
    return [name, properties[name]];
});
```
Since it's just one line (kinda) you can probably inline it with `const propertyPairs = Object.keys(...)...` in SimpleTable.jsx instead of having it in its own file.

Reply:
:rocket: 28ed46c

Reply:
:rocket: 28ed46c